packages/agent/src/agents/supervisor/supervisorAgent.ts

```
@@ -193,6 +197,30 @@ export class SupervisorAgent extends BaseAgent {
     }
   }
 
+  /**
+   * Initializes the DocumentAgent component if enabled
+   * @param agentConfig - Agent configuration
+   * @private
+   */
+  private async initializeDocumentAgent(
+    agentConfig: AgentConfig | undefined
+  ): Promise<void> {
+    const docCfg = (agentConfig as any)?.documents;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)